include/rtc/description.hpp

```diff
@@ -32,7 +32,7 @@
 
 namespace rtc {
 
-class Description {
+class RTC_CPP_EXPORT Description {
```

Comment:
Same here, you also need the << operators at the end.

Reply:
I only added RTC_CPP_EXPORT to the ones that caused an error to my project. I'll update based on your choices soon!